table/table.go

```diff
@@ -100,15 +97,30 @@ func (t *Table) SelectBuilder(columns ...string) *qb.SelectBuilder {
 
 // Insert returns insert all columns statement.
 func (t *Table) Insert() (stmt string, names []string) {
-	return t.insert.stmt, t.insert.names
+	return t.InsertBuilder().ToCql()
+}
+
+// InsertBuilder returns a builder initialised to insert all columns statement.
+func (t *Table) InsertBuilder() *qb.InsertBuilder {
```

Comment:
The builders are mutable you cannot return a pointer like that.

It should be implemented as UpdateBuilder and DeleteBuilder.